### PR TITLE
Adds HEAD endpoints to high-volume URIs.

### PR DESCRIPTION
**Pull request by SteversIO**
## NOTEworthy

This PR is against an existing feature branch, not our primary branch (`master`).  This is for code review purposes only. Will re-target `master` once the existing feature branch is merged.

## Background

Adds explicit HEAD endpoints for high-volume URIs.

## Changes

Adds HEAD endpoint for the following three high-volume URIs:

- `/`
- `/metadata/...`
- `/image/...`

## Code Review

This Pull Request was thoroughly reviewed and meets all Code Review Standards pertaining to:

- [ ] **Implementation** - satisfied acceptance criteria
- [ ] **Communication** - notified engineers/stakeholders about impactful changes
- [ ] **Error handling** - handled, logged & alerted on errors
- [ ] **Documentation** - wrote readable code & commits, documented unintuitive code
- [ ] **Testing** - unit & E2E test coverage, tested in staging, DB migrations backwards compatible
- [ ] **Security** - sanitized inputs, vetted dependencies, validated/secured API requests, no committed secrets
- [ ] **Performance** - optimized expensive algorithms & database queries

Please select all **applied** standards relevant to this PR.
## Confirmation

- [ ] **Assignee Confirmation** - I (the author) have filled out the template above
- [ ] **Reviewer Confirmation** - I (a/the reviewer) confirm 1) the author has filled out the template and checked the box that says **Assignee Confirmation** 2) I reviewed the code and selected all applicable items in the code review section.

### CODE REVIEW

**Review by tunguyennnnn — APPROVED** [review]
LGTM

File: src/controllers/MetaDataController.test.ts [hunk 1]
```diff
@@ -53,6 +53,38 @@ describe('MetaDataController', () => {
...
   });
 
+  describe('HEAD', () => {
+    let dummyDomainName: string;
+
+    beforeEach(() => {
+      dummyDomainName = 'unstoppable.crypto';
+    });
+
+    describe('/metadata/:DomainOrToken', () => {
```
Comment (DChan0319) [inline src/controllers/MetaDataController.test.ts:63]: ":domainOrToken"

**Review by DChan0319 — APPROVED** [review]
Looks good to me~!

File: src/controllers/MetaDataController.test.ts (hunk 1 above)
Comment (tunguyennnnn) [reply src/controllers/MetaDataController.test.ts:63]: oh yea good catch 👍

**Comment by SteversIO** [comment]
Thanks all. I'll correct the capitalization 💪

**Pull request merged**